src/modules/txbuilder/RadixTransactionBuilder.ts

```diff
@@ -376,6 +381,104 @@ export default class RadixTransactionBuilder {
         return this
     }
 
+    /**
+     * Mint and transfer a multi-issuance token in one atom.
+     * 
+     * @param ownerAccount The owner of the token.
+     * @param tokenReference Token reference.
+     * @param decimalQuantity The amount to mint and transfer.
+     * @param from Sender account, needs to have RadixAccountTransferSystem.
+     * @param to Receiver account
+     * @param message Optional message.
+     */
+    public mintAndTransferTokens(
+        ownerAccount: RadixAccount,
+        tokenReference: string | RRI,
+        decimalQuantity: string | number | Decimal,
+        from: RadixAccount,
+        to: RadixAccount,
+        message?: string,
+    ) {
+
+        tokenReference = (tokenReference instanceof RRI)
+            ? tokenReference
+            : RRI.fromString(tokenReference)
+
+        const tokenClass = ownerAccount.tokenDefinitionSystem.getTokenDefinition(tokenReference.getName())
+
+        if(!tokenClass) {
+            throw new Error(`ERROR: Token ${tokenReference.getName()} not found.`)
+        }
+
+        const subunitsQuantity = this.getSubUnitsQuantity(decimalQuantity)
+
+        if (subunitsQuantity.lt(this.BNZERO)) {
+            throw new Error('Negative quantity is not allowed')
+        } else if (subunitsQuantity.eq(this.BNZERO)) {
+            throw new Error(`Quantity 0 is not valid`)
+        }
+
+        if (tokenClass.tokenSupplyType !== RadixTokenSupplyType.MUTABLE) {
+            throw new Error('This token is fixed supply')
+        }
+
+        if (subunitsQuantity.gte(tokenClass.getUnallocatedSupply())) {
+            throw new Error('Total supply would exceed 2^256')
+        }
+
+        if (from.address.equals(to.address)) {
+            throw new Error(`Cannot send money to the same account`)
+        }
+
+        const unallocatedTokens = tokenClass.getUnallocatedTokens()
+        const tokenPermissions = unallocatedTokens[0].getTokenPermissions()
+
+
+        const particleGroup = new RadixParticleGroup()
+        const consumerQuantity = new BN(0)
+        for (const consumable of unallocatedTokens) {
+            particleGroup.particles.push(RadixSpunParticle.down(consumable))
+
+            consumerQuantity.iadd(consumable.getAmount())
+            if (consumerQuantity.gte(subunitsQuantity)) {
+                break
+            }
+        }
+
+        // Remainder
+        if (consumerQuantity.sub(subunitsQuantity).gtn(0)) {
+            particleGroup.particles.push(RadixSpunParticle.up(
+                new RadixUnallocatedTokensParticle(
+                    consumerQuantity.sub(subunitsQuantity),
+                    tokenClass.getGranularity(),
+                    Date.now(),
+                    tokenReference,
+                    tokenPermissions,
+                )))
+        }
+
+        const particle = new RadixTransferrableTokensParticle(
```

Comment:
I'm a little surprised this works, I though we would have to create a TransferrableTokens particle in the minter account, then instantly down it and create a new one in the receiving account. Very good that it does though, this is more efficient